spec/models/provider_spec.rb

```diff
@@ -268,7 +268,8 @@
   end
 
   describe "courses" do
-    let(:course) { TestSetup.unpub_pri_math }
+    let(:test_setup) { TestSetup.new }
+    let(:course) { test_setup.get(:course, :primary, :unpublished) }
```

Comment:
I think rename `test_setup` to `test_data`

Reply:
👍